packages/legacy/core/App/onboarding.ts

```diff
@@ -63,6 +68,7 @@ export const generateOnboardingWorkflowSteps = (
     isPrefaceComplete(didSeePreface, showPreface ?? false),
     isOnboardingTutorialComplete(didCompleteTutorial),
     isTermsComplete(Number(didAgreeToTerms), termsVersion),
+    isAttemptLockoutComplete(servedPenalty),
```

Comment:
Order matters here somewhat, no? Would it be better just above `isAuthenticationComplete(didCreatePIN, didAuthenticate)`?